client/js/models/Header.js

```diff
@@ -1,9 +1,3 @@
-module.exports = Object.assign({}, require('./__proto__.js'), {
-    data: [
-        'about us',
-        'where to find us',
-        'future days farm',
-        'the blog',
-        'our offerings'
-    ]
-})
+module.exports = { ...require('./__proto__.js'),
+    data: ['about us', 'where to find us', 'future days farm', 'the blog', 'our offerings']
+}
```

Comment:
I think your data is too minimal here, which forces you to do a bunch of extra stuff ( `capitalizeWords` function, the regex replace on click) later on to compensate. Try an array of objects with a name/label organization:
`data: [ { name: 'about-us', label: 'About Us' }, etc. ]`
Then in the template, you can iterate over the data and add a `data-name` attribute and use the label for the text content:
`const navOptions = model.map(datum => `<div data-name="${datum.name}" data-js='navList'>${datum.label}</div>`).join('')`
The `data-name` can then be used to retrieve the appropriate route when you click on the nav item.

Reply:
I'm struggling with the markup here with the template strings. Apologies while I figure that out.

Reply:
Also, it would be more semantic to use list elements rather than divs for the nav items.

Reply:
I am kind of okay with the minimal data here to be honest.  Scott, I think your proposed changes are fine too, although I am not a fan of employing `data` elements that linger in the DOM.

It would be cool to upgrade the `View` module to automatically bind navigation to `data-link`, which could be removed from the DOM, but this presents other concerns.